3-platform/warehouse/db_postgres/src/main/resources/db/changelog/changes/01-create-warehouses-table.sql

```diff
@@ -0,0 +1,15 @@
+-- warehouses table
+
+CREATE TABLE public.warehouses
+(
+    id      varchar NOT NULL,
```

Comment:
NOT NULL for Primary key? could it have been null?

Reply:
the same for FK

Reply:
BTW: foreign key could be null for some cases. 

e.g. inventory -> product. 

Inventory is allocated, but product is not assigned.

Reply:
I think it's better to leave that not null constraint for PK, just as agreement that every field should have that parameter (null/not null)